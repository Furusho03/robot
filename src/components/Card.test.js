import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card';

import toJson from 'enzyme-to-json';
// import TestRenderer from 'react-test-renderer';

// const CardRender = TestRenderer.create(
//     <Card />
// );

it('test render', () => {  
    expect(toJson(shallow(<Card />))).toMatchSnapshot();
})
