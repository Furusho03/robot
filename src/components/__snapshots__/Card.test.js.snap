// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render 1`] = `
<div
  className="tc bg-light-green dib br3 ma2 grow bw2 shadow-S"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
